graftm/pplacer.py

```diff
@@ -3,12 +3,22 @@
 import json
 import timeit
 import logging
+import time
 
 from Bio import SeqIO
 
 from graftm.classify import Classify
 from graftm.housekeeping import HouseKeeping
 
+def timeit(method):
```

Comment:
this method is used in both hmmer and pplacer (at least). Don't repeat yourself. Make a new class that has this method and use that class from both places, perhaps make it a staticmethod for ease of use.